lute/require/src/bundlevfs.cpp

```diff
@@ -0,0 +1,127 @@
+#include "lute/bundlevfs.h"
+
+#include "Luau/Common.h"
+
+#include <string>
+
+static const std::string BUNDLE_PREFIX = "@bundle/";
+
+BundleVfs::BundleVfs(const Luau::DenseHashMap<std::string, std::string>& bundleMap)
+    : filePathToBytecode(bundleMap)
+{
+}
```

Comment:
```suggestion
BundleVfs::BundleVfs(Luau::DenseHashMap<std::string, std::string> bundleMap)
    : filePathToBytecode(std::move(bundleMap))
{
}
```
I think it makes sense for BundleVfs to own this? (Unrelated to `isBundleModule` and `isBundleDirectory` taking `bundleMap` as a reference.) Would also need to change the member's type to a value instead of a reference.